firmware/2/releases.json

```diff
@@ -1,4 +1,19 @@
 [
+    {
+        "required": false,
+        "version": [2, 7, 2],
+        "min_bridge_version": [2, 0, 7],
+        "min_firmware_version": [2, 0, 8],
+        "min_bootloader_version": [2, 0, 0],
+        "bootloader_version": [2, 1, 4],
+        "translations": ["cs-CZ", "de-DE", "es-ES", "fr-FR"],
+        "url": "data/firmware/2/trezor-2.7.2.bin",
+        "url_bitcoinonly": "data/firmware/2/trezor-2.7.2-bitcoinonly.bin",
```

Comment:
i'm thinking we should just point to `t2t1` directory here, perhaps even make this `releases.json` a straight up copy of the `t2t1` version ... and then not place new releases in the `2/` subdir.

what do you think @komret @vdovhanych ?

Reply:
This directory is not accessed by Suite, not even old versions, AFAIK. I suppose old versions of trezorctl use `releases.json` from here? If that is the only use case, then it could indeed point to the `t2t1` directory instead.

Reply:
and yet, you merged with this thing untouched :) well there's always next time i suppose

Reply:
I'll do a separate PR.
@vdovhanych, could we just redirect from `https://data.trezor.io/firmware/2/releases.json` to `https://data.trezor.io/firmware/t2t1/releases.json`? Or do we have to maintain a copy of the file?

Reply:
Here it is: https://github.com/trezor/data/pull/80